File: dubbo-service-csharp/trunk/java-hessian-server/src/main/java/com/eqying/pf/service/provider/api/UserServiceI.java
package com.eqying.pf.service.provider.api;

import com.eqying.pf.service.provider.model.User;

/**
 * TODO(这个类的作用)
 *
 * @auther: renjunjie
 * @since: 2016/12/5 17:07
 */
public interface UserServiceI {

	public User getUserInfo(String userId);

	public int saveUser(User u);

	public int updateUser(User u);

	public int deleteUser(String userId);

}

File: dubbo-service-demo/dubbo-demo/Service/UserServiceI.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using com.eqying.pf.service.provider.model;

namespace com.eqying.pf.service.provider.api
{
    public interface UserServiceI
    {
        User getUserInfo(String userId);
    }
}
